modules/monitoring-moving-monitoring-components-to-different-nodes.adoc

```diff
@@ -52,37 +71,112 @@ metadata:
   namespace: openshift-monitoring
 data:
   config.yaml: |
-    *prometheusOperator:
+    prometheusOperator:
       nodeSelector:
-        foo: bar
+        nodename: controlplane1
     prometheusK8s:
       nodeSelector:
-        foo: bar
+        nodename: worker1
+        nodename: worker2
     alertmanagerMain:
       nodeSelector:
-        foo: bar
+        nodename: worker1
+        nodename: worker2
+        nodename: worker3
```

Comment:
```nodename: worker3``` could be removed

Reply:
@juzhao thank you. I have now removed that line.